scripts/tests/conftest.py

```diff
@@ -0,0 +1,3 @@
+def pytest_addoption(parser):
```

Comment:
what is this for?

Reply:
As it says in the code: it's needed to placate `setup.cfg`.